test/coverlet.core.tests/Coverage/CoverageTests.cs

```diff
@@ -10,8 +10,7 @@ namespace Coverlet.Core.Tests
 {
     public partial class CoverageTests
     {
-        private readonly InstrumentationHelper _instrumentationHelper = new InstrumentationHelper(new ProcessExitHandler(), new RetryHelper(), new FileSystem());
-        private readonly Mock<ILogger> _mockLogger = new Mock<ILogger>();
```

Comment:
revert and use `_mockLogger` on below tests.

Reply:
Done